Build/Customizations/Version.props

```diff
@@ -9,6 +9,6 @@
                        -x ......... CI, nightly (will be suffixed by '.1.buildnumber')
                        -x ......... local (will be suffixed by '.9.yymmdd-hhMMss')
     -->
-    <Version Condition="'$(Version)' == ''">3.0.0</Version>
+    <Version Condition="'$(Version)' == ''">4.0.0</Version>
```

Comment:
In the ticket it mentions "hotfix" version, here we are releasing a new major version. I assume this is intended?

Reply:
This is a leftover from the release of RPA. As I dont have push rights for develop, the commit was only on my local branch. @MichaelKetting  can you please add the update Metadata commit to develop? Thanks!

Reply:
This has already been resolved, github seems to be bugged, this file should not be shown in this PR anymore as the changes are already included in develop.